.github/workflows/pull-request-title-validation.yml

```diff
@@ -25,12 +25,11 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Validate PR title
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
```

Comment:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable